src/test/java/resources/batch.feature

```diff
@@ -88,8 +88,6 @@ Feature: Batch Details API test
     And I should get batchId in response
     Given I have a batch id from previous scenario
     And I have tenant as "gorilla"
-    When I call the operations-app auth endpoint with username: "mifos" and password: "password"
```

Comment:
These lines have no effect on the test,  is auth auth-enabled check is available inside